### PR TITLE
Add core weekly 51

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2019-12-30-coreweekly-week-51-2019.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 16th to Sunday 22th of December 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear Developers,
+
+Merry Christmas ! Pablo Borowicz, PrestaShop's lead developer, [published the next part](https://build.prestashop.com/news/prestashop-in-2019-and-beyond-part-3-the-future-architecture/) of his [series about the architecture of the PrestaShop open source project](http://build.prestashop.com/news/prestashop-in-2019-and-beyond-introduction/) as a present for you ;) . We hope you like it !
```
Comment (matks) [inline news/_posts/core-weekly/2019-12-30-coreweekly-week-51-2019.md]: @eternoendless please validate 😄

Comment (eternoendless) [reply news/_posts/core-weekly/2019-12-30-coreweekly-week-51-2019.md]:
```suggestion
Happy holidays! Pablo Borowicz, PrestaShop's lead developer, [published the third part](https://build.prestashop.com/news/prestashop-in-2019-and-beyond-part-3-the-future-architecture/) of his [series about the architecture of the PrestaShop open source project](http://build.prestashop.com/news/prestashop-in-2019-and-beyond-introduction/) as a present for you ;) . We hope you will like it!
```

**Review by eternoendless — COMMENTED** [review]
Very small changes needed

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thanks @matks

**Pull request merged**